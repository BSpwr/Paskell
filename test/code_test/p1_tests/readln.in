This
class
is
awesome!
1337