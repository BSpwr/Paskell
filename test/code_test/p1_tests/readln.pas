program ReadlnTest;
var
    a : string;
    b : string;
    c : string;
    d : string;
    e : real;
begin
    readln(a,b,c,d, e);

    writeln(a);
    writeln(b);
    writeln(c);
    writeln(d, ' ', e);
end.