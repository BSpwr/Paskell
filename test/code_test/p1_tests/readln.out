This
class
is
awesome! 1337.0
